lib/shopify_cli/messages/messages.rb

```diff
@@ -415,7 +415,7 @@ module Messages
               Usage: {{command:%s login [--store=STORE]}}
           HELP
           invalid_shop: <<~MESSAGE,
-            Invalid store provided (%s). Please provide the store in the following format: my-store.myshopify.com
+            Invalid store provided (%s). Please make sure that the store belongs to your partner organization, and provide the store in the following format: my-store.myshopify.com
```

Comment:
@MeredithCastile This is the error message shown when either: the store is invalid, or the store doesn't belong to the partners account. I'm not entirely sure if it would be better to separate the cases into two error messages... Open to any suggestions!

Reply:
@molly-yu It would probably be better to separate it into two specific error messages. I am getting this error since upgrading to 2.9.0 even though I have been granted partner access to a store, and I don't know how to fix it.